sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/KnownMethods.cs

```diff
@@ -39,6 +39,7 @@ public static bool IsMainMethod(this IMethodSymbol methodSymbol)
                     (methodSymbol.Parameters.Length == 1 && methodSymbol.Parameters[0].IsType(KnownType.System_String_Array)));
         }
 
+        #region --- Object Methods ---

```

Comment:
I think we might have extracted methods a bit too soon (most of them are only used once and so don't deserve to be put aside for now).